health/health.d/ram.conf

```
@@ -37,6 +37,18 @@
     info: percentage of estimated amount of RAM available for userspace processes, without causing swapping
       to: sysadmin
 
+   alarm: oom_kill
+      on: mem.oom_kill
+      os: linux
+   hosts: *
+  lookup: sum -10m unaligned
+   units: kills
+   every: 10s
+    warn: $this > 0
+   delay: down 1h multiplier 1.5 max 1h
+    info: number of out of memory kills in the last 10 minutes
+      to: sysadmin
```

```suggestion
   alarm: oom_kill
      on: mem.oom_kill
      os: linux
   hosts: *
  lookup: sum -1m unaligned
   units: kills
   every: 30s
    warn: $this > 0
   delay: down 5m
    info: number of out of memory kills in the last 10 minutes
      to: sysadmin
```

Some thoughts:
 - we don't want to lose oom kills (`every` needs to be < then `lookup`)
 - we don't want to have this alarm to be active forever (down 1h is too much, as you know there is no way to _acknowledge_ alarms)
 - since it is sum and warn > 0 we don't want big lookup and small every, because the alarm will be active for the whole lookup period.

What do you think?

---

Discussed and agreed.